webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -170,11 +164,12 @@ export default class AnnouncementBar extends React.PureComponent<Props, State> {
                 css={{gridArea: 'announcement'}}
                 data-testid={this.props.id}
             >
-                <OverlayTrigger
-                    delayShow={Constants.OVERLAY_TIME_DELAY}
+                <WithTooltip
```

Comment:
@M-ZubairAhmed Out of the scope of this PR, and probably something we won't tackle for a while, but I feel the announcement banner is making a weird use of the tooltips. Maybe we should revisit the announcement banner as a whole with UX to clarify how it should look and behave, instead of being so "free" as it is now.

Reply:
@larkox i definetly agree with that do you think we can merge this pr without those change or should we fix that and then come back to this one?

Reply:
I would say let's merge this one, since it is practically done, and we can open a more detailed effort afterwards.

Reply:
Feel free to open a ticket with proposed fix when you get time